test/parallel/test-util-inspect.js

```diff
@@ -1900,3 +1900,9 @@ assert.strictEqual(
     '{ foo: [Getter/Setter] Set { [ [Object], 2, {} ], ' +
       "'foobar', { x: 1 } },\n  inc: [Getter: NaN] }");
 }
+
+{
+  const a = [1, 2, 3];
+  Object.setPrototypeOf(a, Number.prototype);
+  assert.strictEqual(inspect(a), "Number { '0': 1, '1': 2, '2': 3 }");
+}
```

Comment:
can you also add a test for

```js
const a = [1, 2, 3];
Object.setPrototypeOf(a, Object.create(null))
```

Reply:
We already have tests for that :-)

Reply:
@BridgeAR that's not null prototype, its a prototype with a null prototype, and it currently throws.

```
> Object.setPrototypeOf([1, 2, 3], Object.create(null))
TypeError: clazz is not a constructor
    at noPrototypeIterator (internal/util/inspect.js:479:14)
```